app/javascript/member-facing/backbone/action_form.js

```diff
@@ -55,10 +54,8 @@ const ActionForm = Backbone.View.extend({
   //    bucket: if passed, submitted to the server in the form
   //    location: a hash of location values inferred from the user's request
   //    skipPrefill: boolean, will not prefill if true
-  //    consentNeeded: when true and only for new members, the form will trigger a
-  //      validation if the selected country is in the EU and if consent hasnt been
-  //      given. If consent has been given or if the country is not in the EU
-  //      then a regular action is created.
+  //    async: when true the form will validate by default.
```

Comment:
pls pls add a similar comment back explaining the logic of the display hidden element args or what gets displayed in which condition. This GDPR user flow logic is esoteric a-f.